### ISSUE TITLE
Option for excluding catch statements

### ISSUE THREAD

**Issue by richardsimko** [sweepline/eslint-plugin-unused-imports#93]
Catch statements aren't allowed without a variable for the error object, it would be nice to have an option to exclude them from the check so that code like this is allowed:

```ts
try {
    something()
} catch (e) {
    logger.warn('Something failed');
}
  ```

If it makes sense I'm happy to submit a PR, probably after #89 is merged though.

**Comment by antfu** [comment]
This library doesn't really handle where and how unused things are detected, but only wrapping https://eslint.org/docs/latest/rules/no-unused-vars over and provides fixes for imports. The thing you encounter actually comes from `@typescript-eslint`'s change in v8, which I would suggest you report to them first.

**Comment by richardsimko** [comment]
They have the config `{ "caughtErrors": "none" }`, can I use that with unused-imports as well then?

https://eslint.org/docs/latest/rules/no-unused-vars#caughterrors-none

**Comment by richardsimko** [comment]
Thanks!
